docs/conf.py

```
@@ -5,11 +5,13 @@
 # https://www.sphinx-doc.org/en/master/usage/configuration.html
 
 import os
+from datetime import datetime
 
 # -- Project information -----------------------------------------------------
 
 project = "Isaac Teleop"
-copyright = "2025-2026, NVIDIA CORPORATION & AFFILIATES."
+copyright = f"2025-{datetime.now().year}, NVIDIA CORPORATION & AFFILIATES"
```

Ruff (0.15.2)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)